projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/__tests__/deploy-job-component.spec.ts

```diff
@@ -2,14 +2,13 @@
  * Copyright 2023-2023 VMware, Inc.
  * SPDX-License-Identifier: Apache-2.0
  */
-import DeployJobDialog, {
-  IDeployJobDialogProps
-} from '../components/DeployJob';
+import DeployJobDialog from '../components/DeployJob';
 import { render, fireEvent } from '@testing-library/react';
 import { jobData } from '../jobData';
 import { VdkOption } from '../vdkOptions/vdk_options';
+import { IJobNameAndTeamProps, IJobPathProp } from '../components/props';
 
-const defaultProps: IDeployJobDialogProps = {
+const defaultProps: IJobNameAndTeamProps & IJobPathProp = {
```

Comment:
nit: is there reason for 2 interfaces , why not 1  - IJobDetails or something. 

Interface called Field1&Field is a code smell a bit. You could not figure out what is the correct abstraction.

Reply:
I changed the interface into type and created a third type which represents  Field1&Field2